rpm/.gitignore

```diff
@@ -1,2 +1,5 @@
+kube*
```

Comment:
I would go with explicit ignore here, this technically matches several files already in git control.